package.json

```diff
@@ -89,6 +90,7 @@
     "pg": "^8.2.1",
     "postcss-cli": "^7.1.1",
     "postcss-nested": "^4.2.1",
+    "prisma": "^2.16.1",
```

Comment:
`peerOptionalDependencies` still lists `  "@prisma/client": "^2.12.0"`. Should that be updated as well?

Reply:
Hmm yes good catch thanks! Will update.

(At least now the instructions work for the current release of Prisma!)